src/psse_export.jl

```diff
@@ -0,0 +1,1369 @@
+const PSSE_EXPORT_SUPPORTED_VERSIONS = [:v33]  # TODO add :v34
+const PSSE_DEFAULT = ""  # Used below in cases where we want to insert an empty field to signify the PSSE default
+const PSSE_BUS_TYPE_MAP = Dict(
```

Comment:
We should have a map of this kind in PowerSystems.jl to make it consistent with the parser.

Reply:
Yes we should. I suspect there are a lot of things here that should be consolidated with the parser in PSY